api/server/router/system/system_routes.go

```diff
@@ -82,6 +82,9 @@ func (s *systemRouter) getInfo(ctx context.Context, w http.ResponseWriter, r *ht
 			info.OperatingSystem = "<unknown>"
 		}
 	}
+	if versions.LessThan(httputils.VersionFromContext(ctx), "1.41") {
+		info.CgroupVersion = 0
```

Comment:
should we keep it to `1` ?

Reply:
No, old API clients doesn't expect CgroupVersion to exist.